README.asciidoc

```diff
@@ -409,24 +424,62 @@ $ vagrant install-openshift3-assets-base
 
 Given this base foundation, you may want to `vagrant package` the result before proceeding to install OpenShift code.
 
+[source, sh]
 ----
 $ vagrant install-openshift3
 $ vagrant build-openshift3-base-images  # pick up updates if older "deps" base reused
-$ vagrant build-openshift3 --images
+$ vagrant build-openshift3 --images # --images not needed for bootstrap stages
 $ vagrant build-sti --binary-only
 ----
 
 You may again wish to package this in order to reuse.
 
-If your provider is not set up to automatically sync changes from a local,
-you may do the following to get the host prepared for a git-based sync
-and then use that:
+If you want an image with:
+* enabled and running openshift
+* deployed docker registry
+* deployed routing layer
+* Sample project named 'Turbo'
 
+[source, sh]
+----
+$ vagrant bootstrap-openshift3
+----
+
+You should package this image so it will be easier for you to restore to a known state. If you used the +--local-source+ flag during the +origin-init+ this packed image may be shared with others.
+
+If you want to use this image for OpenShift platform development and if your provider is not set up to automatically sync changes from a local, you may do the following to get the host prepared for a git-based sync and then use that:
+[source, sh]
 ----
 $ vagrant clone-upstream-repos  # and then to git sync and build:
 $ vagrant sync-openshift3 -s
 ----
 
+=== Cliff Notes for bootstrap stage vagrant box
+
+Assuming you are targeting the v0.4.4 for your vagrant box
+
+[source, sh]
+----
+$ unset GOPATH
+$ cd ~/tmp
+$ vagrant openshift3-local-checkout --replace  --branch v0.4.4
+$ pushd origin
+$ vagrant origin-init --stage bootstrap --os fedora openshift3-bootstrap --local-source
```

Comment:
Shouldn't you be starting from inst here?  Doesn't make since to start from bootstrap to build bootstrap.